sdks/java/io/kafka/build.gradle

```diff
@@ -148,6 +148,7 @@ kafkaVersions.each {kv ->
       if (!(kv.key in sdfKafkaVersions)) excludeTestsMatching "*DynamicPartitions" //admin client create partitions does not exist in kafka 0.11.0.3 and kafka sdf does not appear to work for kafka versions <2.0.1
       if (!(kv.key in sdfKafkaVersions)) excludeTestsMatching "*SDFResumesCorrectly" //Kafka SDF does not work for kafka versions <2.0.1
       if (!(kv.key in sdfKafkaVersions)) excludeTestsMatching "*StopReadingFunction" //Kafka SDF does not work for kafka versions <2.0.1
+      if (!(kv.key in sdfKafkaVersions)) excludeTestsMatching "*WatermarkUpdateWithSparseMessages" //Kafka SDF does not work for kafka versions <2.0.1
```

Comment:
minor. since there are multiple exclude with same condition.
```
if (!(kv.key in sdfKafkaVersions)) {
  excludeTestsMatching ...
  excludeTestsMatching ...
}
```